pkgs/development/libraries/drogon/default.nix

```diff
@@ -43,6 +43,9 @@ stdenv.mkDerivation rec {
     # this part of the test would normally fail because it attempts to configure a CMake project that uses find_package on itself
     # this patch makes drogon and trantor visible to the test
     ./fix_find_package.patch
+
+    # avoid race condition
+    ./no_parallel_test.patch
```

Comment:
Did `enableParallelChecking = false;` not work?

Reply:
To my understanding `enableParallelChecking` works by passing flags to well-known tools, right?  Drogon uses it's own custom testing framework so I don't think it would have worked.

Reply:
it still seems to fail of OfBorg so maybe my race condition theory is wrong regardless...